configs/projects/puppet-agent.rb

```diff
@@ -71,7 +70,8 @@
     end
   elsif platform.is_windows?
     # For windows, we need to ensure we are building for mingw not cygwin
-    host = "--host #{platform.platform_triple}"
+    platform_triple = platform.platform_triple
+    host = "--host #{platform_triple}"
```

Comment:
But, it was working before, so it's definitely not strictly necessary. I'm not really sure why it was working before...